home/ivp-planning/getting-started/login.md

```diff
@@ -0,0 +1,57 @@
+# Login
+
+{% hint style="info" %}
+For better understanding we created a ![](../../../.gitbook/assets/YouTube\_icon.png) [iVP YouTube Video](https://www.youtube.com/watch?v=iYbY-axZlkE&list=PLlzoGkRUR67houzn5F5ejD3R-kQrDcps5&index=17) where you can find a detailed tutorial for the Login with  a Microsoft Azure e-mail account.
+{% endhint %}
+
+When starting iVP Planning, you will have to log in with your Microsoft-Account.  
+To do so, iVP will open up a web page in your browser that will guide you through the log-in process.  
+
+![](/.gitbook/assets/login-azure-example.png)
+
+The Login requires your Microsoft email adress and your Microsoft password and will show the following notice (in the same browser window) in case of a successful login:
+
+{% hint style="info" %}
+Authentication complete. You can return to the application. Feel free to close this browser tab.
+{% endhint %}
+
+Regardless wether you close the browser window or not, iVP Planning is now ready to use.
+
+# Why do I need to login to iVP Planning?
+
+As part of [Version 2301](../changelog/version-2301) iVP Planning migrated to a User Management.
```

Comment:
Please remove the link to the version. This page doesn't exist yet

Reply:
removed